linera-client/src/client_options.rs

```diff
@@ -1004,6 +1004,16 @@ pub enum NetCommand {
         #[arg(long, num_args=0..=1)]
         binaries: Option<Option<PathBuf>>,
 
+        /// Don't build docker image. This assumes that the image is already built.
+        #[cfg(feature = "kubernetes")]
+        #[arg(long, default_value = "false")]
+        no_build: bool,
+
+        /// The name of the docker image to use.
+        #[cfg(feature = "kubernetes")]
+        #[arg(long, default_value = "linera:latest")]
+        docker_image_name: String,
+
```

Comment:
What's the link with the kubernetes feature and why `--kubernetes` in the title of this PR?

Reply:
`--kubernetes` calls the kubernetes version of `linera net up`, which needs to be compiled with the `kubernetes` feature. But if you don't compile with the `kubernetes` feature, the `--kubernetes` option is not even available.
You can still compile with the `kubernetes` feature and run just `linera net up` without `--kubernetes`, and it won't run the kubernetes version

Reply:
I see. I misread the test plan (which could have mentioned these extra arguments)

Reply:
nit: It's hard to understand the context because the motivation section doesn't mention kubernetes.

Reply:
Good point! I edited the motivation, hopefully it's clearer now